target/linux/ath79/dts/qca9556_avm_fritzdvbc.dts

```diff
@@ -0,0 +1,108 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include <dt-bindings/gpio/gpio.h>
+#include "qca9556_avm_fritz-repeater.dtsi"
+
+/ {
+	compatible = "avm,fritzdvbc", "qca,qca9556";
+	model = "AVM FRITZ!WLAN Repeater DVB-C";
+
+	aliases {
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+	};
+
+	led_spi {
+		compatible = "spi-gpio";
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		sck-gpios = <&gpio 14 GPIO_ACTIVE_LOW>;
+		mosi-gpios = <&gpio 15 GPIO_ACTIVE_LOW>;
+		num-chipselects = <0>;
+
+		spi_gpio: led_gpio@0 {
+			compatible = "fairchild,74hc595";
+			reg = <0>;
+			gpio-controller;
+			#gpio-cells = <2>;
+			registers-number = <2>;
+			spi-max-frequency = <10000000>;
+
+			gpio_latch_bit {
+				gpio-hog;
+				gpios = <15 GPIO_ACTIVE_HIGH>;
+				output-high;
+				line-name = "gpio-latch-bit";
+			};
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_power: power {
+			label = "fritzdvbc:green:power";
+			gpios = <&spi_gpio 5 GPIO_ACTIVE_LOW>;
+		};
+
+		wlan {
+			label = "fritzdvbc:green:wlan";
+			gpios = <&spi_gpio 6 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy1tpt";
```

Comment:
Based on the label, there is only one LED and you are "arbitrarily" selecting one WiFi for it here?
Or is this selection inherited from vendor firmware?

Reply:
Selecting the 2.4 radio as default is commonplace as we can not bind two radio triggers on one LED.

Reply:
I typically prefer to have nothing set in this case, so the user can set it up the way he/she likes. Having LED activity by default would trick the uneducated user to assume vendor behavior (i.e. both WiFi are relevant) is implemented.
However, I'm aware that you could bring up a similar argument for your side, and after all it's not really important and a matter of taste anyway.